ui/src/app/components/resources.tsx

```diff
@@ -3,22 +3,9 @@ import * as React from 'react';
 import {Clickable, MenuItem} from 'app/components/buttons';
 import {ClrIcon} from 'app/components/icons';
 import {PopupTrigger} from 'app/components/popups';
-import colors from 'app/styles/colors';
 import {switchCase} from 'app/utils';
 import {ResourceType} from 'app/utils/resourceActionsReact';
 import {environment} from 'environments/environment';
-import {Cohort, ConceptSet} from 'generated/fetch';
-
-const styles = {
-  listItem: {
-    border: '0.5px solid #C3C3C3', margin: '.4rem',
-    height: '1.5rem', display: 'flex'
-  },
-  listItemCheckbox: {
-    height: 17, width: 17, marginLeft: 10, marginTop: 10,
-    marginRight: 10, backgroundColor: colors.green[1]
-  }
-};
 
 export const ResourceCardMenu: React.FunctionComponent<{
```

Comment:
I wasn't sure if this should fold under resource card now, or continue being here. It is currently only used in one place, but I think will probably fold out again in the next few weeks.